### PR TITLE
Portfolio Popup Window

### PR DESCRIPTION
**Pull request by IfzaRasool**
**Popup window**

> 

- Add EventListener on buttons using Javascript

- Refactor markup using Javascript

- Add Functionality on popup close icon

### CODE REVIEW

**Review by MrBrN197 — COMMENTED** [review]
## Hi, @IfzaRasool  @steveWDamesJr  👋 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a javascript project, that does not have ESlint linters setup
please use the correct linter setup.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -309,9 +164,37 @@ <h1 class="contact-heading">CONTACT US</h1>
...
+        <li class="modal-divisor"><img src="Icons/Rectangle.png" alt="divide line2"></li>
+        <li>Ruby on rails</li>
+        <li class="modal-divisor"> <img src="Icons/Rectangle.png" alt="divide line3"></li>
+        <li>css</li>
+        <li class="modal-divisor"><img src="Icons/Rectangle.png" alt="divide line3"></li>
+    </ul>
+
+<form class="modal-form">
+<button class="modal-button" type="submit">See Live</button>
+<button class="modal-button" type="submit">See Source</button>
```
Comment (ShoiraTa) [inline index.html:190]: - [ ] Please add the icons on buttons as given in Figma, match colors and make sure that text is not wrapping 😉 : 

Yours  | Figma
------------- | -------------
image | image

File: index.html [hunk 2]
```diff
@@ -309,9 +164,37 @@ <h1 class="contact-heading">CONTACT US</h1>
...
             </footer>
         </section>
-        
     </main>
+
+    <div class="modal-container">
+    <div class="modal">
+        <div class="modal-item">
+        <h1 class="modal-heading">Multi-post story</h1>
+        <img  class="modal-close" src="Icons/modal-close.png" alt="close">
```
Comment (ShoiraTa) [inline index.html:173]: - [ ] I kindly suggest you to delete the extra margin to properly align items on modal, to compare:

Before  | After
------------- | -------------
image  | image

You can also see that before, `x` button is too close to the right border, to fix this you can set the  `justify-content: space-around;` 😉 
If you apply these changes, your design will be the same as given in Figma 👍🏻

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @IfzaRasool ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Keep on coding, good job so far! :clap:

There are some minor issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi again @IfzaRasool ,  

Now everything looks great!
Your project is complete! Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**